examples/siro_sandbox/sandbox_app.py

```diff
@@ -194,10 +210,41 @@ def visualize_task(self):
                     #     goal_position + box_size / 2,
                     #     color
                     # )
+                    if isinstance(self.gui_agent_ctrl, GuiHumanoidController):
+                        # draw can place area
+                        can_place_position = goal_position.copy()
+                        can_place_position[1] = self.get_agent_feet_height()
+                        self._debug_line_render.draw_circle(
+                            can_place_position,
+                            1,
+                            mn.Color3(255 / 255, 255 / 255, 0),
+                            24,
+                        )
+
+                    assert self._held_target_obj_idx is not None
+                    if self.gui_input.get_key_down(GuiInput.KeyNS.SPACE):
+                        translation = self.get_agent_translation()
+                        dist_to_obj = np.linalg.norm(
+                            goal_position - translation
+                        )
+                        if dist_to_obj < self._can_grasp_place_threshold:
+                            self._held_target_obj_idx = None
+                            # object_drop_height = 0.15
+                            drop_pos = (
+                                goal_position
+                                # + mn.Vector3(0, object_drop_height, 0)
+                            )
+
             except ValueError:
                 pass
```

Comment:
What's the need for this? If a held object is not in `idxs`?

Reply:
Good catch! Legacy code, can be removed.